routes/rooms.go

```diff
@@ -149,7 +193,7 @@ func DeleteEmergencyRoom(ctx *gin.Context) {
 	db := models.GetDB(logCtx)
 
 	erIdRaw := ctx.Param("id")
-	log.Debug().Str("requested_id", erIdRaw)
+	log.Debug().Str("delete_id", erIdRaw)
```

Comment:
.Send missing

Reply:
This is missing in every single endpoint? Why so?

Reply:
negligence

Reply:
fixed